lib/MetaCPAN/Script/CVE.pm

```diff
@@ -0,0 +1,249 @@
+package MetaCPAN::Script::CVE;
+
+use Moose;
+use namespace::autoclean;
+
+use Cpanel::JSON::XS          qw( decode_json );
+use Log::Contextual           qw( :log :dlog );
+use Path::Tiny                qw( path );
+use Ref::Util                 qw( is_arrayref );
+use MetaCPAN::Types::TypeTiny qw( Bool Str Uri );
+use MetaCPAN::Util            qw( numify_version );
+
+use Data::Dumper;    # REMOVE
```

Comment:
Should this be removed?

Reply:
yes. it will be removed when the script starts upserting to ES